### DISCUSSION TITLE
Suggestion for the Blocked Services

### DISCUSSION THREAD

**Discussion by Leurie02** [AdguardTeam/AdGuardHome#4319] [Ideas]
Please add a Button for add services and also... Please add the blocking for games like...
PUBG
ML
COC
COD
DOTA
LOL
and more games...

Please... more games blocking.... thank you so much and God Bless...

**Comment by Aikatsui** [comment]
I think that's good to accomplish using https://github.com/AdguardTeam/AdGuardHome/issues/2230 (because there're so many games) but game distribution services (e.g. steam) / studios can be added to https://github.com/AdguardTeam/AdGuardHome/issues/2604
